### PR TITLE
GH-1200 Adds params to Grid designMode styles

### PR DESCRIPTION
**Pull request by jbrundage**
Partially Fixes GH-1200

http://rawgit.com/jbrundage/Visualization/Grid_Imps/demos/dermatology.html?src/layout/Grid

Signed-off-by: jay <jbrundage372@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage Looks fine - if you fix the lint issue and remove the commented out CSS code we can pull?

File: src/layout/Grid.css [hunk 1]
```diff
@@ -15,7 +15,7 @@
 .layout_Grid .dropCell {
     box-sizing:border-box;
     background-color: lightgrey;
-    opacity: 0.33;
+    /*opacity: 0.33;*/
```
Comment (GordonSmith) [inline src/layout/Grid.css]: remove commented code

File: src/layout/Grid.css [hunk 2]
```diff
@@ -31,19 +31,24 @@
 }
 .layout_Grid .dragItem { 
     z-index: 0;
-    opacity:0.33;
+    /*opacity:0.33;*/
```
Comment (GordonSmith) [inline src/layout/Grid.css]: remove commented code

File: src/layout/Grid.css [hunk 3]
```diff
@@ -31,19 +31,24 @@
...
     opacity: 1;
 }
+.layout_Grid .layout_Surface{
+    height: 100%;
+    width: 100%;
+    box-sizing:border-box;
+}
 .layout_Grid div[draggable=true] { 
-    opacity: 0.75;
+    /*opacity: 0.75;*/
```
Comment (GordonSmith) [inline src/layout/Grid.css]: remove commented code

File: src/layout/Grid.css [hunk 4]
```diff
@@ -31,19 +31,24 @@
...
     cursor: move;
 }
-.layout_Grid div[draggable=true] > .layout_Surface{ 
+/*.layout_Grid div[draggable=true] > .layout_Surface{
     overflow:visible;
-}
+}*/
```
Comment (GordonSmith) [inline src/layout/Grid.css]: remove commented code

**Comment by GordonSmith** [comment]
@jbrundage Still some commented code to remove.

**Comment by GordonSmith** [comment]
@jbrundage bump

**Comment by jbrundage** [comment]
@GordonSmith Should be good to go now

**Comment by GordonSmith** [comment]
@jbrundage Failing unit tests...

**Comment by jbrundage** [comment]
@GordonSmith Sorry about that... it should be resolved now and ready to pull

**Comment by mlzummo** [comment]
looks good

**Pull request merged**